includes/wc-functions.php

```diff
@@ -288,16 +288,15 @@ function ( $value ) {
     if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) {
         $manage_stock = 'no';
         $backorders   = 'no';
-        $stock        = '';
         $stock_status = wc_clean( $data['_stock_status'] );
-
+    
```

Comment:
Could you please remove the extra spaces?

Reply:
Done